src/Autonomous.cpp

```diff
@@ -558,7 +558,7 @@ void Autonomous::autonomyIter()
   if (USE_MAP) {
       if (mapLoopCounter++ > mapBlindPeriod) {
           map.addPoints(toTransform(pose), lidar_scan, mapDoesOverlap ? 0.4 : 0);
-          mapDoesOverlap = lidar_scan.size() > mapOverlapSampleThreshold;
```

Comment:
Perhaps a cleaner way to fix this is change `mapOverlapSampleThreshold` to an unsigned int in `Autonomous.h`

Reply:
Yeah, makes sense. Better to change the variable itself if it doesn't need to be signed